README.md

```diff
@@ -1 +1,173 @@
+  <ul>
+    <li><a href="https://restcountries.com/">RestCountries API</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>none</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Countries List**
+- **Country Details**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://country-details-five.vercel.app/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: Node.js, NPM on your machine.
+
+### Setup
+
+Clone this repository to your desired folder: by using this command: 
+```bash
+git clone https://github.com/MuhammadUsmanAslam/country-details.git
+```
+
+### Install
+
+Install this project with command: 
+```bash
+npm i
+```
+
+### Usage
+
+To run the project, execute the following command: 
+```bash
+npm run start
+```
+
+### Test
+
+To run unit tests for this the project, execute the following command: 
+```bash
+npm run test
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author: Muhammad Usman Aslam**
+
+- GitHub: [@githubhandle](https://github.com/MuhammadUsmanAslam)
+- Twitter: [@twitterhandle](https://twitter.com/M_Usman_Aslam)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/muhammad-usman-aslam)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **List of Countries displayed on home page**
+- **Single List item details**
+- **Unit testing with jest**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+- If you like this project you can share this on your social media.
+- If there is any issue there you can open new issue on the repository.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Orignal design was from <a href="https://www.behance.net/sakwadesignstudio">Nelson Sakwa</a>
+
+I would also like to thank the <a href="https://stackoverflow.com/">stackoverflow</a> community whenever I got stuck into a problem or error got my solution there.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly your `MIT` License link is not working, please create it and replace the link to correct one 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/233703806-ee640a2e-7f16-4b74-8754-d2c3194b9352.png)